src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,8 @@
 
 -->
 ## Upcoming Release
+* Edited `New-AzVM` to create a new storage account for boot diagnostics if one does not exist. 
+  This will prevent the cmdlet from using a random storage account in the current subscription to use for boot diagnostics.
```

Comment:
```suggestion
* Updated `New-AzVM` to create a new storage account for boot diagnostics if one does not exist. This will prevent the cmdlet from using a random storage account in the current subscription to use for boot diagnostics.
```